common.gypi

```diff
@@ -2,14 +2,15 @@
   'target_defaults': {
     'default_configuration': 'Release',
     'cflags_cc' : [
-      '-std=c++14',
-      '-Wall',
-      '-Wextra',
-      '-ffunction-sections -fdata-sections',
-      '-D_GLIBCXX_USE_CXX11_ABI=0',
-    ],
-    'ldflags': [
-      '-Wl,--gc-sections'
+      '-std=c++11',
```

Comment:
Why C++11 here but above we use C++14?

Reply:
The `common.gypi` is a reusable set of defaults for `node-gyp` projects from the `node-cpp-skel` project.  Values here are intended to be overridden by more specific values in `binding.gyp`, which is what this code now does.